source/playbooks/SC/ssmdocs/scripts/sc_get_input_values.py

```diff
@@ -3,6 +3,15 @@
 
 SC_mappings = {
     "CloudWatch.1": {
+        "filter_name": "RootAccountUsage",
+        "filter_pattern": '{$.userIdentity.type="Root" && $.userIdentity.invokedBy NOT EXISTS && $.eventType !="AwsServiceEvent"}',
+        "metric_name": "RootAccountUsage",
+        "metric_value": 1,
+        "alarm_name": "RootAccountUsage",
+        "alarm_desc": "Alarm for RootAccountUsage > 0",
+        "alarm_threshold": 1
```

Comment:
Given that I don't fully understand the meaning of these values they may already be completely correct, but given that the alarm is supposed to trigger at `> 0` should both `metric_value` and `alarm_threshold` = `1` here, or should one or both of them be `0` for correct alarm behavior?

Reply:
I believe these values are derived from the controls: https://docs.aws.amazon.com/securityhub/latest/userguide/cloudwatch-controls.html under the 'To create a metric filter and alarm' sections.